Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -165,11 +172,11 @@ void HiresTexture::Prefetch()
 				// But bad luck, SOIL isn't, so TODO: remove SOIL usage here and use libpng directly
 				// Also TODO: remove s_textureCacheAquireMutex afterwards. It won't be needed as the main mutex will be locked rarely
 				//lk.unlock();
-				HiresTexture* t = Load(base_filename, 0, 0);
+				std::unique_ptr<HiresTexture> texture = Load(base_filename, 0, 0);
```

Comment:
I think that's less flexible overall than returning a unique_ptr. It also doesn't really make sense, considering shared semantics only really begin for the texture instance once it enters the cache.